src/app/src/interfaces/IMovie.ts

```diff
@@ -0,0 +1,17 @@
+export interface IMovie {
```

Comment:
the file name should include model `Imovie.model.ts` so it's easier to distinguish that it is an interface. We also store them in `models` folder. Might be a bit confusing but that is the way we do it in the project. In p9 you might find some old places with `someName.int.ts` but we try to not add new stuff like that